### PR TITLE
Add keyboard: takayoshiotake/five_octave

### PR DESCRIPTION
**Pull request by takayoshiotake**
Add keyboard takayoshiotake/five_octave

### CODE REVIEW

**Comment by takayoshiotake** [comment]
I also added a default keymap.

File: keyboards/takayoshiotake/five_octave/keymaps/vial/vial.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
+{
+    "name": "Takayoshi Otake Five Octave VIAL",
+    "vendorId": "0x1209",
+    "productId": "0x8CAE",
```
Comment (lesshonor) [inline keyboards/takayoshiotake/five_octave/keymaps/vial/vial.json]:
```suggestion
```
These attributes are not used by Vial.

File: keyboards/takayoshiotake/five_octave/info.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x8CAE",
+        "vid": "0x1209",
+        "force_nkro": true
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
```
Comment (lesshonor) [inline keyboards/takayoshiotake/five_octave/info.json]: These settings are not enabled by default, so there is no need to explicitly disable them here.
```suggestion
```

File: keyboards/takayoshiotake/five_octave/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+		_______, _______, _______, _______, _______, _______
+	),
+
+	[3] = LAYOUT(
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______
+	),
```
Comment (lesshonor) [inline keyboards/takayoshiotake/five_octave/keymaps/default/keymap.c]: No need to hardcode filler `KC_TRNS` layers. For non-dynamic keymaps it's pointless, and dynamic keymaps will automatically fill in "missing" layers at compile-time up to the maximum set in `dynamic_keymap.layer_count`. (By default, this is 4.)
```suggestion
    )
```

File: keyboards/takayoshiotake/five_octave/keymaps/vial/keymap.c [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+		_______, _______, _______, _______, _______, _______
+	),
+
+	[3] = LAYOUT(
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+		_______, _______, _______, _______, _______, _______
+	),
```
Comment (lesshonor) [inline keyboards/takayoshiotake/five_octave/keymaps/vial/keymap.c]:
```suggestion
)
```
Please see previous comment about extra blank layers.

**Review by lesshonor — CHANGES_REQUESTED** [review]

**Comment by takayoshiotake** [comment]
Thank you for your response. I applied the suggestions and cleaned up the code.

**Review by lesshonor — APPROVED** [review]

**Pull request merged**